modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/rebalance/DistributionZoneRebalanceEngine.java

```diff
@@ -251,7 +251,7 @@ private CompletableFuture<Void> recalculateAssignmentsAndTriggerZonePartitionsRe
             HybridTimestamp timestamp,
             int catalogVersion
     ) {
-        return distributionZoneManager.dataNodes(timestamp, catalogVersion, zoneDescriptor.id())
+        return distributionZoneManager.dataNodes(catalogVersion, zoneDescriptor.id())
```

Comment:
I thought we have agreed in private that we still need timestamp here, something has changed?